### PR TITLE
Changes to navbar

### PR DESCRIPTION
**Pull request by udayk8139**
- Added text-shadow to all the header text so it's readable when images
are below.
- Changed the navbar color to FCC's primary colour, and used white to
highlight the navigation menu and the active elements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
 
     <!-- Theme CSS -->
-    <link href="css/agency.min.css" rel="stylesheet">
+    <link href="css/agency.css" rel="stylesheet">
```
Comment (vramana) [inline index.html]: @udayk8139 Why change this? Any reasons? If not, revert this.

File: index.html [hunk 2]
```diff
@@ -78,9 +78,9 @@
     <header>
         <div class="container">
             <div class="intro-text">
-                <!--<div class="intro-lead-in">Welcome To Our Studio!</div>
```
Comment (vramana) [inline index.html]: Rather Uncommenting you should delete this or write something more appropriate.

Comment (udayk8139) [inline index.html]: I thought they can be useful...

File: index.html (hunk 1 above)
Comment (udayk8139) [inline index.html]: Because I can't change the .min.css files, I need to edit the .css files first and then minifiy them up.

**Pull request closed without merging**